packages/js/product-editor/src/components/attribute-term-input-field/attribute-term-input-field.tsx

```diff
@@ -172,6 +208,7 @@ export const AttributeTermInputField: React.FC<
 							return {
 								...changes,
 								inputValue: state.inputValue,
+								isOpen: true,
```

Comment:
Any specific reason you added this, this causes some weird behaviour when editing an attribute. As the dropdown is automatically open already. I would suggesting removing this.

Reply:
Yes, I added it to show the spinner while the term is being created. Otherwise, the user won't know if anything is happening underneath until the new term is created.
We also have a problem with the `focus`, since after pressing `+ Create "[term-name]"` the focus goes back to the input, so the list of terms should be visible.

![Screen Capture on 2023-08-09 at 10-14-50](https://github.com/woocommerce/woocommerce/assets/1314156/b30265f1-6efc-4e2a-964f-4b2a3b56681e)

Reply:
Gotcha, that makes sense, although I think it is still weird behaviour to have it auto open in the edit menu.
Could we disable it there?
https://github.com/woocommerce/woocommerce/assets/2240960/282fbd5a-9c3d-455f-8ca8-1c3a37a44a10